cmd/gossamer/prune_test.go

```diff
@@ -6,36 +6,35 @@ import (
 	"testing"
 
 	"github.com/dgraph-io/badger/v2"
-
 	"github.com/stretchr/testify/require"
 )
 
-func iterateDB(db *badger.DB, cb func(*badger.Item)) {
+func iterateDB(db *badger.DB, cb func(*badger.Item)) { //nolint
```

Comment:
what's the lint error?

Reply:
unused, since i skipped the test that uses it, since it's failing on missing db keys :/ we need to re-generate the db to fix it, or move this test the state package